renderer/styles/style.css

```diff
@@ -869,12 +871,10 @@ span.qc-option-txt{
   padding: 0 0 20px 0;
 }
 .mcq-popup-actions {
-  position: fixed;
-  bottom: 0;
   text-align: center;
-  padding: 3.7%;
   width: 96vw;
   border-top: 1px solid rgba(34, 36, 38, .15);
+  padding: 12px;
```

Comment:
Why are you using some random number to fix the issue. Fix at the root.

Reply:
I have given the padding based on button height